src/Randomizer.SMZ3.Tracking/Configuration/Yaml/BCU/responses.yml

```diff
@@ -3,6 +3,12 @@
   Weight: 0.1
 - Text: My fate is sealed. I have <break strength='weak'/> no choice.
   Weight: 0.5
+- Text: This uh <break time='100ms'/> next band asked me not to read this but goddamn it, I'm going to read it anyway, because I wrote it, and it's the truth. I fucking love this band! They are the best band ever, period! Ladies and gentlemen, Tracker!
```

Comment:
Most of these aren't really BCU specific. This one is a Tenacious D reference, the other is Skyrim and there's a Portal one.

Reply:
Admittedly part of this is because a friend of mine plays rando and has a kid so perhaps I'm influenced by that a bit, but part of what I was doing was pulling out stuff with curse words and stuff from the sassy tracker to keep it more "PG." Sorry, I thought I kinda mentioned that, but I suppose I didn't like go too in depth into what I meant there.

Stashing these in the BCU one doesn't necessarily feel "right" either tbh, but I didn't know if we'd want to juggle 3 (sort of 4 with the generics) sets of built in configs.

Reply:
I don't think this would be a bad reason to add an extra profile, though.

Reply:
Alright, I can split it out, then. I kinda wanna name it "Rated T for Teen" but maybe that's too much of an American game industry reference. lol

Reply:
It gets the point across and it's a bonus sneaky BCU reference in a way, too.